v3/otm_adapt.cpp

```diff
@@ -317,15 +318,19 @@ bool otm_adapt(const input& in, state& s)
   interpolate_point_data(a, s.K);
   interpolate_point_data(a, s.G);
   interpolate_point_data(a, s.rho);
-  interpolate_point_data(a, s.ep);
+  if (s.ep.size() > 0) interpolate_point_data(a, s.ep);
```

Comment:
Is there a better criterion that we could add to the input data to establish whether or not we should expect ep to be allocated?

Reply:
@japlews I guess that since we don't expect to have many material models, a workable criterion could be what kind of material model we are using. This could be problematic when having many material models, but I don't think that would be the case for this code.

Reply:
But I guess that brings the issue of how we deal with multiple materials. Something that will have to wait until we know if we got more funding for this.